packages/fuchsia_ctl/lib/src/ssh_client.dart

```diff
@@ -31,6 +31,7 @@ class SshClient {
     String targetIp,
     List<String> command = const <String>[],
   }) {
+    assert(command != null);
```

Comment:
does this execute with asserts enabled?

Reply:
Just for tests.

If this were to get through at runtime we'd get a null ref exception with the stack pointing to the bad use.  I guess this could be a check and throw instead but it seems ok.